src/rantanplan/core.py

```diff
@@ -725,8 +725,20 @@ def generate_liaison_positions(syllables, liaison):
     liaison_indices = [
         index for index, position in enumerate(positions) if position
     ]
+    # Priorize single liaisons
```

Comment:
"Priorize" does not exist, "Prioritize" does.

Reply:
I think that ideally it should inverse rank by the number of consecutive `1`'s instead of just prioritizing the single ones.